src/ensembl/src/styles/main.scss

```diff
@@ -5,7 +5,7 @@
 
 html,
 body,
-.ens-app {
+[class='ens-app'] {
```

Comment:
Because it will get a css module suffix (unless there is a way to prevent webpack from adding suffixes to specific classes). And the html element with this class name is in the template html file, not in a React component; so in order to target this element the class name has to stay unmodified.